data/102/080/273/102080273.geojson

```diff
@@ -117,12 +120,12 @@
         "spa",
         "eng"
     ],
-    "wof:lastmodified":1566600191,
+    "wof:lastmodified":1684438911,
     "wof:name":"Culebra",
-    "wof:parent_id":85687331,
+    "wof:parent_id":"85633729",
```

Comment:
Oops, parent_id should be an int!

Reply:
Fixed by https://github.com/whosonfirst-data/whosonfirst-data-admin-pr/pull/9/commits/a069b8b6c49c7aa471c0c592d4c2ce543bfa1832.